src/events/ready.js

```diff
@@ -1,34 +1,18 @@
-const { Attachment } = require('discord.js');
 const config = require('../../config');
-const { getReservedImage } = require('../utils');
 
 const currentDate = new Date(Date.now()).toLocaleString('PT');
 
 module.exports = async client => {
-  const helloImage = new Attachment(getReservedImage('hello'));
-  await client.channels
-    .get(config('bot.mainChannelId'))
-    .send(config('messages.enter'), {
-      files: [helloImage],
-    });
-
   const { user } = client;
 
   await user.setActivity(config('activities.enter'));
   await user.setStatus('online');
 
   process.on('SIGINT', async () => {
-    await user.setStatus('dnd');
-    await user.setActivity(config('activities.exit'));
-
-    const goodbyeImage = new Attachment(getReservedImage('goodbye'));
-    const messageSent = await client.channels
-      .get(config('bot.mainChannelId'))
-      .send(config('messages.exit'), {
-        files: [goodbyeImage],
-      });
+    const newStatus = await user.setStatus('dnd');
```

Comment:
Does the Discord library have constants for these values? It would read better.

Reply:
Will check.

Reply:
I haven't found a way.

Reply:
Fuck it, then.